From 03a8751e91c4f08e0f9ae6ff906237e51b9d8f0e Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Wed, 18 Nov 2015 19:09:39 +0200
Subject: [PATCH 0548/1240] fix: ddr: a3700: Upgrade DRAM init in NTIM to v7
 script

- Update the DDR init procedure to be in sync with version 7
  of XDB script
- The values for DDR init are received from DDR support team
  after tests and should improve the DRAM stability

Change-Id: Ia886389a4a6e3e8a1ef9bdd6aa62427157ddcb7d
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/25007
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Nadav Haklai <nadavh@marvell.com>
(cherry picked from commit 13b44e0208e90b8275edb369ed040901901765e8)
Reviewed-on: http://vgitil04.il.marvell.com:8080/25355
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 tools/wtp/u-boot-ntim.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/wtp/u-boot-ntim.txt b/tools/wtp/u-boot-ntim.txt
index 5fe0c15..a9b958a 100644
--- a/tools/wtp/u-boot-ntim.txt
+++ b/tools/wtp/u-boot-ntim.txt
@@ -75,7 +75,7 @@ WRITE: 0xC0000304 0x0
 WRITE: 0xC0000308 0x0
 WRITE: 0xC0000200 0xe0001
 WRITE: 0xC0000204 0x0
-WRITE: 0xC0000220 0x16020532
+WRITE: 0xC0000220 0x13020532
 WRITE: 0xC0000044 0x30200
 WRITE: 0xC00002c0 0x6000
 WRITE: 0xC00002c4 0x00100020
@@ -127,7 +127,7 @@ WRITE: 0xC00010a4 0x0
 WRITE: 0xC00010a8 0x0
 WRITE: 0xC00010ac 0x0
 WRITE: 0xC00010b0 0x0
-WRITE: 0xC0001000 0x4040
+WRITE: 0xC0001000 0x44041
 WRITE: 0xC0001014 0x80200
 WRITE: 0xC0001038 0x2
 WRITE: 0xC000103c 0x10
-- 
1.9.1

